docs/en/sdk-manual/web.md

```diff
@@ -0,0 +1,236 @@
+# Clickstream Web SDK
+
+## Introduction
+
+Clickstream Web SDK can help you easily collect click stream data from browser to your AWS environments through the data pipeline provisioned by this solution.
+
+The SDK is based on the amplify-js SDK core library and developed according to the amplify-js SDK plug-in specification. In addition, the SDK provides features that automatically collect common user events and attributes (for example, screen view, and first open) to accelerate data collection for users.
+
+## Integrate the SDK
+
+### Include SDK
+
+```bash
+npm install @aws/clickstream-web
+```
+
+### Initialize the SDK
+
+You need to configure the SDK with default information before using it. Copy your configuration code from your clickstream solution control plane, the configuration code should look like as follows. You can also manually add this code snippet and replace the values of appId and endpoint after you registered app to a data pipeline in the Clickstream Analytics solution console.
```

Comment:
`control plane` -> `web console` for consistency

Reply:
changed.